<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GbsTableWidget</class>
 <widget class="QWidget" name="GbsTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>共找到数据:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labTotal">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labUnit">
       <property name="text">
        <string>条</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>当前页数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labPage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetPageBtn" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="btnFirst">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>首页</string>
           </property>
           <property name="text">
            <string>首页</string>
           </property>
           <property name="icon">
            <iconset resource="../main.qrc">
             <normaloff>:/image/首页.png</normaloff>:/image/首页.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnPrevious">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>上一页</string>
           </property>
           <property name="text">
            <string>上一页</string>
           </property>
           <property name="icon">
            <iconset resource="../main.qrc">
             <normaloff>:/image/上一页 .png</normaloff>:/image/上一页 .png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnNext">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>下一页</string>
           </property>
           <property name="text">
            <string>下一页</string>
           </property>
           <property name="icon">
            <iconset resource="../main.qrc">
             <normaloff>:/image/下一页.png</normaloff>:/image/下一页.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnLast">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="toolTip">
            <string>尾页</string>
           </property>
           <property name="text">
            <string>尾页</string>
           </property>
           <property name="icon">
            <iconset resource="../main.qrc">
             <normaloff>:/image/尾页.png</normaloff>:/image/尾页.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main.qrc"/>
 </resources>
 <connections/>
</ui>
